microk8s-resources/actions/enable.kubeflow.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source $SNAP/actions/common/utils.sh
+
+
+CONTROLLER="uk8s"
+CLOUD="microk8s"
+MODEL="kubeflow"
+
+
+function print_message () {
+
+cat << EOF
+
+Congratulations, Kubeflow is now available.
+Run \`microk8s.kubectl proxy\` to be able to access the dashboard at
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/#!/overview?namespace=$MODEL
+
+The central dashboard is available at http://$1/
+To tear down Kubeflow and associated infrastructure, run this command:
+
+    microk8s.juju kill-controller $CONTROLLER --destroy-all-models --destroy-storage
+
+For more information, see documentation at:
+https://github.com/juju-solutions/bundle-kubeflow/blob/master/README.md
+
+EOF
+
+}
+
+
+function enable_addons () {
+  "$SNAP/microk8s-enable.wrapper" dns storage dashboard juju
```

Comment:
This could probably get changed to the bash equivalent of this:
https://github.com/juju-solutions/bundle-kubeflow/blob/master/scripts/deploy-microk8s#L62
(I've since updated the scripts to use Python instead of Bash, but the logic remains the same)

Reply:
Using a python script to enable kubeflow is fine since we will end up including python in the snap for the clustering work. What is the best way to consume your scripts? I do not see versioned releases so should we copy them over to MicroK8s whenever we want to have an add-on release? Should we pull the scripts whenever the `microk8s.enable kubeflow` runs? Should we have a pip release? Something else?

I would rather not have and maintain a bash recreation of your work. Since you are making a kubeflow installer lets use that directly.

Reply:
@ktsakalozos: Probably the easiest way at this point would be to just clone this repo and run `./scripts/deploy-microk8s {create|destroy}`. The charms won't work independently yet due to needing resources set up that Juju doesn't directly support yet.